libraries/standard/mqtt/test/unit/iot_tests_mqtt_platform.c

```diff
@@ -0,0 +1,306 @@
+/*
+ * IoT MQTT V2.1.0
+ * Copyright (C) 2018 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
Wondering what the rule we follow is for this date, as some files in the repo are 2018 and others are 2019. Should this be 2020?

Reply:
Updated.